flink-core/src/main/java/org/apache/flink/configuration/CheckpointingOptions.java

```diff
@@ -39,8 +39,12 @@
      * StateBackendFactory#createFromConfig(ReadableConfig, ClassLoader)} method is called.
      *
      * <p>Recognized shortcut names are 'hashmap' and 'rocksdb'.
+     *
+     * @deprecated Use {@link StateBackendOptions#STATE_BACKEND}.
      */
-    @Documentation.Section(value = Documentation.Sections.COMMON_STATE_BACKENDS, position = 1)
+    @Documentation.Section(value = Documentation.Sections.COMMON_STATE_BACKENDS)
+    @Documentation.ExcludeFromDocumentation("Hidden for deprecated")
+    @Deprecated
     public static final ConfigOption<String> STATE_BACKEND =
```

Comment:
Should we change this to ` public static final ConfigOption<String> STATE_BACKEND = StateBackendOptions.STATE_BACKEND`, to deduplicate?

Reply:
I think it's okay to refactor like this to deduplicate.
BTW. @carp84 also suggested this change offline yesterday but agreed that such change is not something must do.

Reply:
I'll create a hotfix to resolve this if plan to merge this PR today once a CI turned green.